src/assets/icons/misc/ArrowSquareOut.svg

```diff
@@ -0,0 +1,5 @@
+<svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M12.1875 16.875L16.8744 16.8744L16.875 12.1875" stroke="white" stroke-width="1.5" stroke-linecap="round" stroke-linejoin="round"/>
+<path d="M11.2478 11.2478L16.8728 16.8728" stroke="white" stroke-width="1.5" stroke-linecap="round" stroke-linejoin="round"/>
+<path d="M8.75 14.375H3.75C3.58424 14.375 3.42527 14.3092 3.30806 14.1919C3.19085 14.0747 3.125 13.9158 3.125 13.75V3.75C3.125 3.58424 3.19085 3.42527 3.30806 3.30806C3.42527 3.19085 3.58424 3.125 3.75 3.125L13.75 3.125C13.9158 3.125 14.0747 3.19085 14.1919 3.30806C14.3092 3.42527 14.375 3.58424 14.375 3.75V8.75" stroke="white" stroke-width="1.5" stroke-linecap="round" stroke-linejoin="round"/>
+</svg>
```

Comment:
The assets file name should start with small letters i.e, `arrowSquareOut.svg` instead `ArrowSquareOut.svg`.